motoman_msgs/srv/ReadSingleIO.srv

```diff
@@ -0,0 +1,3 @@
+uint32 address
+---
+int32 value
```

Comment:
It would be nice if we could add some comments to this message (and its write counterpart) which provide info on which addresses can actually be read from (all *valid* addresses) and which can be written to (see @ted-miller's comment).

Reply:
Assuming here you mean [this comment](https://github.com/ros-industrial/motoman/pull/355#issuecomment-683132425). @ted-miller do the valid addresses for all I/O start at `xxx10` rather than `xxxx1`? I seem to remember that being true for our universal inputs too

Reply:
> do the valid addresses for all I/O start at xxx10 rather than xxxx1? I seem to remember that being true for the universal inputs on our robot too

Yes.

Reply:
I just added a comment to the WriteSingleIO.srv definition describing the available addresses (using the information from [this comment](https://github.com/ros-industrial/motoman/pull/355#issuecomment-683132425)). However, I'm having trouble understanding which addresses are valid for reading?

Reply:
Essentially *all* addresses are readable, as long as they are valid addresses.

Even though that doesn't really say too much, I still feel like a small comment saying something to that effect will help avoid at least *some* confusion. We can always say something like "Refer to the relevant documentation from Yaskawa for information on which addresses are valid addresses on a specific controller with a specific IO configuration".